PacketDotNet/ASExternalLSA.cs

```diff
@@ -0,0 +1,112 @@
+using System;
```

Comment:
Can you add the license header and copy the copyright from the original file for each of the new files? That way it should obvious to anyone using any file what license is in effect.

Reply:
Otherwise the splitting up looks good. The original file must have been huge and probably should have been broken up a while ago.